content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -46,6 +46,41 @@ check the documentation for that version.
 
 
 <!-- body -->
+## Install and configure prerequisites
+
+These instructions apply common settings for Kubernetes nodes on Linux.
+If you are certain that your setup does not require a specific setting you can skip it.
+Consult the individual container runtime documentation or the
+[Network Plugin Requirements](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements)
+page for further details.
+
+### Forwarding IPv4 and letting iptables see bridged traffic
+
+Make sure that the `br_netfilter` module is loaded. This can be done by running `lsmod | grep br_netfilter`. To load it explicitly call `sudo modprobe br_netfilter`.
+
+As a requirement for your Linux Node's iptables to correctly see bridged traffic, you should ensure `net.bridge.bridge-nf-call-iptables` is set to 1 in your `sysctl` config, e.g.
+
+```bash
+   cat <<EOF | sudo tee /etc/modules-load.d/k8s.conf
+   overlay
+   br_netfilter
+   EOF
+
+   sudo modprobe overlay
+   sudo modprobe br_netfilter
+
+   # Setup required sysctl params, these persist across reboots.
+   cat <<EOF | sudo tee /etc/sysctl.d/k8s.conf
+   net.bridge.bridge-nf-call-iptables  = 1
+   net.bridge.bridge-nf-call-ip6tables = 1
+   net.ipv4.ip_forward                 = 1
+   EOF
+
+   # Apply sysctl params without reboot
+   sudo sysctl --system
+```
+
+For more details please see the [Network Plugin Requirements](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements) page.
```

Comment:
```suggestion
```

we can remove this line, since it's already mentioned here:
https://github.com/kubernetes/website/pull/32946/files#diff-e56be5b5f714d216a9a064b71a8b18655179b1f397f57a2e0dc4f76f12e46958R54